generators/entity-client/files-angular.cjs

```diff
@@ -177,7 +177,7 @@ async function writeAngularFiles() {
     sections: angularFiles,
     rootTemplatesPath: 'angular',
     transform: !application.enableTranslation ? [replaceAngularTranslations] : undefined,
-    context: { ...application, ...entity },
+    context: { ...application, ...entity, entity },
```

Comment:
Not quite happy with this one, please remove one or the other parameter

Reply:
@Tcharl, thanks a lot for taking the time to review this.
I need to pass the entity for some recursive code. the ...entity, is used everywhere else to be able to access fields, relationships... directly without `entity.X` . I really think this is the best approach, but maybe you have a better suggestion